wagtail/contrib/forms/views.py

```diff
@@ -111,69 +109,6 @@ def get_context_data(self, *, object_list=None, **kwargs):
         return context
 
 
-class DeleteSubmissionsView(TemplateView):
-    """Delete the selected submissions"""
-
```

Comment:
I think we should keep the delete submission view. 

This would be a breaking change for some cases (e.g. ModelAdmin Usage).

Reply:
I was trying to write the test cases but sure, I will try to make a PR keeping those classes and we can then update the test cases.